.github/workflows/deploy.yml

```
@@ -32,6 +32,9 @@ jobs:
       - name: Use Node.js
         uses: actions/setup-node@v3
       - run: npm ci
+      - run: npm fetch
```

Should this be `npm run fetch`?

```suggestion
      - run: npm run fetch
```